Tests/ExperimentalTableDrivenSwiftProtobufTests/unittest.pb.swift

```diff
@@ -0,0 +1,4920 @@
+  #elseif _pointerBitWidth(_32)
+    static let _protobuf_messageLayout = SwiftProtobuf._MessageLayout("\00\0\0\u{4}\0\0\0\0\0\u{5}\0\0\u{1}\0\0\0\0\u{14}\0\0{\u{7f}\0\0\u{5}\u{2}\0\0\0\0$\0\0{\u{7f}\0\0\u{9}\u{3}\0\0\0\0\u{18}\0\0{\u{7f}\0\0\u{b}\u{4}\0\0\0\0\u{1c}\0\0{\u{7f}\0\0\u{a}")
+  #else
+    #error("Unsupported platform")
+  #endif
+
+  static func _protobuf_submessage(for token: SwiftProtobuf._MessageLayout.SubmessageToken) -> any SwiftProtobuf.Message.Type {
+    preconditionFailure("invalid submessage token; this is a generator bug")
+  }
+
+  public var isInitialized: Bool {
+    fatalError("table-driven isInitialized not yet implemented")
+  }
+
+  mutating func decodeMessage<D: SwiftProtobuf.Decoder>(decoder: inout D) throws {
+    fatalError("table-driven decodeMessage not yet implemented")
+  }
+
+  func traverse<V: SwiftProtobuf.Visitor>(visitor: inout V) throws {
+    fatalError("table-driven traverse not yet implemented")
+  }
+
+  static func ==(lhs: SwiftProtoTesting_TestOneof, rhs: SwiftProtoTesting_TestOneof) -> Bool {
+    fatalError("table-driven == not yet implemented")
+  }
+}
+
+extension SwiftProtoTesting_TestOneof.FooGroup: SwiftProtobuf.Message, SwiftProtobuf._MessageImplementationBase, SwiftProtobuf._ProtoNameProviding {
+  static let protoMessageName: String = SwiftProtoTesting_TestOneof.protoMessageName + ".FooGroup"
+  static let _protobuf_nameMap = SwiftProtobuf._NameMap(bytecode: "\0\u{2}\u{5}a\0\u{1}b\0")
+  #if _pointerBitWidth(_64)
+    static let _protobuf_messageLayout = SwiftProtobuf._MessageLayout("\0 \0\0\u{2}\0\0\0\0\0\u{1}\0\0\u{5}\0\0\0\0\u{4}\0\0\0\0\0\0\u{5}\u{6}\0\0\0\0\u{10}\0\0\u{1}\0\0\0\u{9}")
+  #elseif _pointerBitWidth(_32)
+    static let _protobuf_messageLayout = SwiftProtobuf._MessageLayout("\0\u{18}\0\0\u{2}\0\0\0\0\0\u{1}\0\0\u{5}\0\0\0\0\u{4}\0\0\0\0\0\0\u{5}\u{6}\0\0\0\0\u{c}\0\0\u{1}\0\0\0\u{9}")
+  #else
+    #error("Unsupported platform")
+  #endif
+
+  static func _protobuf_submessage(for token: SwiftProtobuf._MessageLayout.SubmessageToken) -> any SwiftProtobuf.Message.Type {
+    preconditionFailure("invalid submessage token; this is a generator bug")
+  }
+
+  public var isInitialized: Bool {
+    fatalError("table-driven isInitialized not yet implemented")
+  }
+
+  mutating func decodeMessage<D: SwiftProtobuf.Decoder>(decoder: inout D) throws {
+    fatalError("table-driven decodeMessage not yet implemented")
+  }
+
+  func traverse<V: SwiftProtobuf.Visitor>(visitor: inout V) throws {
+    fatalError("table-driven traverse not yet implemented")
+  }
+
+  static func ==(lhs: SwiftProtoTesting_TestOneof.FooGroup, rhs: SwiftProtoTesting_TestOneof.FooGroup) -> Bool {
+    fatalError("table-driven == not yet implemented")
+  }
+}
+
+extension SwiftProtoTesting_TestRequiredOneof: SwiftProtobuf.Message, SwiftProtobuf._MessageImplementationBase, SwiftProtobuf._ProtoNameProviding {
+  static let protoMessageName: String = _protobuf_package + ".TestRequiredOneof"
+  stat
```

Comment:
Does a default impl for things like this make sense to we only have to generate it if it is needed?  Would that be any cost savings, or does that force runtime costs?  (I guess always generating it could force constant string data sizes also in other places)